pontoon/base/utils.py

```diff
@@ -680,8 +680,7 @@ def is_same(same_translations, can_translate):
         if st.approved and not st.fuzzy:
             return True
     else:
-        if not st.fuzzy:
-            return True
+        return True
 
     return False
```

Comment:
It will, if `can_translate` is `True` and `st.approved` is `False`. In reality that means that a privileged user adds a suggestion that already exists

Reply:
Right, sorry!